front/app/containers/MainHeader/Components/DesktopNavItems/index.tsx

```diff
@@ -67,15 +70,17 @@ const DesktopNavItems = () => {
               />
             );
           }
-
-          return (
-            <DesktopNavbarItem
-              linkTo={linkTo}
-              onlyActiveOnIndex={onlyActiveOnIndex}
-              navigationItemTitle={navigationItemTitle}
-              key={i}
-            />
-          );
+          if (linkTo) {
```

Comment:
When can `linkTo` be `null`?

Reply:
When the visibility rules kick in and the user does not actually have access to this project. Then the link is `null` and we don't render the nav bar item.